### PR TITLE
Update README with parameterized query instructions

### PR DESCRIPTION
**Pull request by wchieng**
The driver has been updated to use Athena's parameterized query feature. This PR updates the README with instructions on how to use that feature.

### CODE REVIEW

**Review by ld-ahartmann — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -473,16 +473,26 @@ Sample output:
 2
 ```
 
-You can also use the `?` syntax with `DB.Query()` or `DB.Exec()` directly.
+### Parameterized Queries
+
+Athena supports parameterized queries (https://docs.aws.amazon.com/athena/latest/ug/querying-with-prepared-statements.html).
```
Comment (ember-stevens) [inline README.md]:
```suggestion
Athena supports parameterized queries: https://docs.aws.amazon.com/athena/latest/ug/querying-with-prepared-statements.html.
```

File: README.md [hunk 2]
```diff
@@ -473,16 +473,26 @@ Sample output:
...
+Parameterized queries allow for re-running the same query with different parameter values at runtime, and help guard 
+against SQL injection attacks. This is especially useful if some of your parameter values are derived from user input.
+
+To use parameterized queries, use `?` as placeholders in the query you're passing to `DB.Query()` or `DB.Exec()`.
```
Comment (ember-stevens) [inline README.md]:
```suggestion
To use parameterized queries, use `?` as placeholders in the query you pass to `DB.Query()` or `DB.Exec()`.
```

**Review by ember-stevens — APPROVED** [review]
LGTM, just a couple of optional suggestions.

**Pull request merged**